tests/h/services/search_index/_queue_test.py

```diff
@@ -300,14 +300,10 @@ def test_if_there_are_multiple_jobs_with_the_same_annotation_id(
 
         counts = queue.sync(len(jobs))
 
-        # Unfortunately the method gets the metrics wrong here: it reports that
-        # it synced two annotations because it processed two different jobs.
-        # But actually the two jobs were for the same annotation and it only
-        # synced one.
         assert counts == {
-            Queue.Result.SYNCED_MISSING.format(tag="test_tag"): 2,
-            Queue.Result.SYNCED_TAG_TOTAL.format(tag="test_tag"): 2,
-            Queue.Result.SYNCED_TOTAL: 2,
+            Queue.Result.SYNCED_MISSING.format(tag="test_tag"): 1,
+            Queue.Result.SYNCED_TAG_TOTAL.format(tag="test_tag"): 1,
+            Queue.Result.SYNCED_TOTAL: 1,
```

Comment:
There is already a test, further up in the file, for what happens when there are two jobs with the same annotation ID and that annotation is already in Elasticsearch: it deletes both jobs, and reports _two_ jobs completed to New Relic.